src/Interpreters/Set.cpp

```diff
@@ -402,8 +402,8 @@ void Set::checkTypesEqual(size_t set_type_idx, const DataTypePtr & other_type) c
                         + data_types[set_type_idx]->getName() + " on the right", ErrorCodes::TYPE_MISMATCH);
 }
 
-MergeTreeSetIndex::MergeTreeSetIndex(const Columns & set_elements, std::vector<KeyTuplePositionMapping> && index_mapping_)
-    : indexes_mapping(std::move(index_mapping_))
+MergeTreeSetIndex::MergeTreeSetIndex(const Columns & set_elements, std::vector<KeyTuplePositionMapping> && indexes_mapping_)
+    : all_key(set_elements.size() == indexes_mapping_.size()), indexes_mapping(std::move(indexes_mapping_))
```

Comment:
Let's initialize each member on a separate line

Reply:
Interesting. I though we need to obey what clang-format says for new code.

Reply:
I think in that case this affects readability of the code. Also you can find similar code in the code base.

Reply:
But we have a lot of these styles https://github.com/ClickHouse/ClickHouse/blob/master/src/Interpreters/Set.h#L35